src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -91,6 +92,12 @@ public class RazorViewEngineOptions
         public IList<string> AreaViewLocationFormats { get; } = new List<string>();
 
         /// <summary>
+        /// The <see cref="MetadataReference" /> instances that should be included in Razor compilation, along with
```

Comment:
Gets !!! I made a suggestion and expected you to do a little something w/ it :smiling_imp: